app/src/interfaces/presentation-links/presentation-links.vue

```diff
@@ -115,19 +127,23 @@ function getRelatedFieldsFromTemplates() {
 
 <template>
 	<div class="presentation-links">
-		<v-button
-			v-for="(link, index) in linksParsed"
-			:key="index"
-			class="action"
-			:class="[link.type]"
-			:secondary="link.type !== 'primary'"
-			:icon="!link.label"
-			:href="link.href"
-			:to="link.to"
-		>
-			<v-icon v-if="link.icon" left :name="link.icon" />
-			<span v-if="link.label">{{ link.label }}</span>
-		</v-button>
+		<template v-for="(link, index) in linksParsed" :key="index">
+			<v-button
+				v-if="link.actionType !== 'flow' || activeFlows.includes(link.flow!)"
```

Comment:
See the other [comment](https://github.com/directus/directus/pull/26022#discussion_r2490665215).

```suggestion
				v-if="link.actionType !== 'flow' || isActiveFlow(link.flow!)"
```